tests/numpy/unop_test.py

```diff
@@ -14,7 +14,7 @@ def usubtest(A: dace.int64[5, 5], B: dace.int64[5, 5]):
 
 
 @dace.program
-def nottest(A: dace.bool[5, 5], B: dace.bool[5, 5]):
+def nottest(A: dace.bool_[5, 5], B: dace.bool_[5, 5]):
```

Comment:
I still don't understand the difference between dace.bool and dace.bool_. Could you please explain?

Reply:
`dace.bool` is supposed to be `bool` or `numpy.bool`. However, this never works for arrays. Any array of type `np.bool` automatically becomes `np.bool_`. This is standard NumPy behavior because they cannot create arrays with types (Python int) that do not have a fixed number of bytes.